cli/azd/pkg/output/ux/endpoint.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package ux
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/azure/azure-dev/cli/azd/pkg/contracts"
+	"github.com/azure/azure-dev/cli/azd/pkg/output"
+)
+
+type Endpoint struct {
+	Endpoint string `json:"Endpoint"`
```

Comment:
To be usable, I think we'd at least need to least list the `serviceResourceId` tied to this endpoint.